typespec-tests/src/main/java/com/encode/bytes/models/Base64UrlArrayBytesProperty.java

```diff
@@ -27,13 +27,16 @@ public final class Base64UrlArrayBytesProperty {
      * @param value the value value to set.
      */
     @Generated
-    @JsonCreator(mode = JsonCreator.Mode.DISABLED)
-    public Base64UrlArrayBytesProperty(@JsonProperty(value = "value") List<byte[]> value) {
+    public Base64UrlArrayBytesProperty(List<byte[]> value) {
         this.value = value.stream().map(el -> Base64Url.encode(el)).collect(java.util.stream.Collectors.toList());
     }
 
     @Generated
-    private Base64UrlArrayBytesProperty() {}
+    @JsonCreator
+    private static Base64UrlArrayBytesProperty fromJson(@JsonProperty(value = "value") List<Base64Url> value) {
+        return new Base64UrlArrayBytesProperty(
+                value.stream().map(el -> el.decodedBytes()).collect(java.util.stream.Collectors.toList()));
+    }
```

Comment:
Use factory method (the `static` method), so that we now can do "overload" on same `List<>` to the ctor.